examples/gno.land/r/demo/multitxtest/multitx_testing.go

```diff
@@ -0,0 +1,62 @@
+package multitxtest
```

Comment:
Should these tests be contained in the `examples` subdirectory, instead of `gnovm`, since these are `.go` files after all?

Reply:
But can you merge the two files into a single `_test.go` file instead of creating this API which won't be reused elsewhere, please?